lib/HTTP/Headers/Util.pm

```diff
@@ -5,7 +5,7 @@ use warnings;
 
 our $VERSION = '6.30';
 
-use base 'Exporter';
+use Exporter 5.57 'import';
```

Comment:
Just a style question here. I know in our other LWP things we tend to:

```perl
use Foo::Bar qw(thing_to_import);
```
https://github.com/libwww-perl/libwww-perl/blob/master/lib/LWP/UserAgent.pm#L16 for example.

Since we already have the version required for `Exporter` in our build, should this instead be

```perl
use Exporter qw(import);
```

Reply:
Right, that would be in line with other uses. I can occasionally run https://metacpan.org/pod/perlimports on the repo to enforce style. That will rewrite imports. It will preserve the version number, though.